providers/cloudflare.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cloudflare",
```

Comment:
question: Have you considered providing a service-specific provider definition, e.g. `cloudflare-turnstile`?

Reply:
Yes, I have considered this option, but since OneSDK supports selecting the service in the map, I believe that having a `cloudflare` provider is more appropriate. Do you see any drawbacks to this solution?

Reply:
For one thing, it's recommended by conventions (see google-apis-*).

The biggest drawback is that you haven't covered all Cloudflare APIs and can't cover the future ones. And you know how problematic it is to change published providers.
However, I can't say if that's still a relevant issue given the new OneSDK.

Reply:
Thanks for explaining it. I changed the provider name to `cloudflare-turnstile` which also solves the problem with default service in previous comment.